hoomd/hpmc/integrate.py

```diff
@@ -311,28 +314,37 @@ def counters(self):
 
 
 class Sphere(_HPMCIntegrator):
-    R""" HPMC integration for spheres (2D/3D).
+    R""" Hard particle Monte Carlo integration method for spheres.
 
-    Args:
-        seed (int): Random number seed
-        d (float): Maximum move displacement, Scalar to set for all types, or a
-          dict containing {type:size} to set by type.
-        a (float): Maximum rotation move to set for all types, or a dict
-          containing {type:size} to set by type.
-        move_ratio (float, only used with **orientable=True**): Ratio of
-          translation moves to rotation moves. (added in version 2.3)
-        nselect (int): The number of trial moves to perform in each cell.
-        deterministic (bool): Make HPMC integration deterministic on the GPU
+    Sphere parameters:
 
-    Hard particle Monte Carlo integration method for spheres.
+    shape (particle type, dict): defines the shape of the object.
 
-    Sphere parameters:
+        Keys:
+            * *diameter* (float, **required**) - diameter of the sphere
+              (distance units)
+            * *ignore_statistics* (**default: False**) - set to True to ignore
```

Comment:
Could we have something like `* *ignore_statistics* (type, **default:** value)

Reply:
Done.